Project.toml

```diff
@@ -17,9 +17,14 @@ InitialValues = "0.2.10, 0.3"
 SplittablesBase = "0.1.11"
 Transducers = "0.4.55"
 julia = "1"
+Folds = "=0.2.6"
+FLoops = "=0.1.11"
```

Comment:
hm, current Floops is v0.2.1, was it installing that version before and causing problems?

Reply:
Seems so

Reply:
Actually, no. These were the versions when it was failing
```
  [7d9f7c33] Accessors v0.1.32
  [e30172f5] Documenter v0.27.24
⌅ [cc61a311] FLoops v0.1.12
  [41a02a25] Folds v0.2.8
  [9c68100b] FoldsThreads v0.1.2-DEV `~/work/FoldsThreads.jl/FoldsThreads.jl`
  [069b7b12] FunctionWrappers v1.1.3
  [22cec73e] InitialValues v0.3.1
  [171d559e] SplittablesBase v0.1.15
  [28d57a85] Transducers v0.4.76
  [8dfed614] Test `@stdlib/Test`
```

Reply:
Which is worrying that a breakage happened in a patch version of Folds or FLoops